<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>honeymoonplus</class>
 <widget class="QDialog" name="honeymoonplus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">


        Light  #4D545B #505F69 (unpressed)

        Normal #31363B #32414B (border, disabled, pressed, checked, toolbars, menus)

        Dark   #232629 #19232D (background)

</string>
  </property>
  <widget class="QTabWidget" name="honeymoon">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>1041</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter honeymoon</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>lieu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>225</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ID0">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_lieu1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prix1">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color: #505F69;

  border: 1px solid #32414B;

  color: #F0F0F0;

  border-radius: 4px;

  padding: 3px;

  outline: none;

}

</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color: #505F69;

  border: 1px solid #32414B;

  color: #F0F0F0;

  border-radius: 4px;

  padding: 3px;

  outline: none;

}

</string>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>56</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>nom hotel</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nomhotel_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher honeymoon</string>
    </attribute>
    <widget class="QTableView" name="table_honeymoon">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>691</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_par_lieu">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par lieu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recherche_honeymoon">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lieu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prix</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_par_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri_par_prix">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Tri par prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_par_prix">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_par_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_par_lieu">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Modifier et Supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mettre le ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lieu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit_blasa">
     <property name="geometry">
      <rect>
       <x>302</x>
       <y>240</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit_soum">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sauvegrader">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>400</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color: #505F69;

  border: 1px solid #32414B;

  color: #F0F0F0;

  border-radius: 4px;

  padding: 3px;

  outline: none;

}

</string>
     </property>
     <property name="text">
      <string>Sauvegarder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_esm">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: yellow;
    selection-background-color: darkgray;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>nom hotel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="get_data_by_id">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {

  background-color: #505F69;

  border: 1px solid #32414B;

  color: #F0F0F0;

  border-radius: 4px;

  padding: 3px;

  outline: none;

}

</string>
     </property>
     <property name="text">
      <string>Recherche par ID</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
